src/components/ClusterManager.js

```diff
@@ -363,6 +413,17 @@ export default ajaxCaller(class ClusterManager extends PureComponent {
     this.resetUpdateInterval()
   }
 
+  async componentDidUpdate(prevProps) {
+    const prevCluster = _.last(_.sortBy('createdDate', _.remove({ status: 'Deleting' }, prevProps.clusters))) || {}
+    const cluster = this.getCurrentCluster() || {}
+
+    if (cluster.status === 'Error' && prevCluster.status !== 'Error' && !_.includes(cluster.id, errorNotifiedClusters.get())) {
+      notify('error', 'Error Creating Cluster', {
```

Comment:
We need to update the atom here, or we'll keep getting notifications for the same cluster. Something like:
```
errorNotifiedClusters.update(Utils.append(cluster.id))
```